.vale/fixtures/RedHat/ConfigMap/testvalid.adoc

```diff
@@ -2,4 +2,5 @@ valid content: config map
 valid content: config maps
 valid content: configure
 valid content: kubeconfig
-
+It uses the install-config file.
```

Comment:
For body text, this form is invalid, it must be enclosed in backticks. Can we limit this to headings only?